rides/templates/extend/email.html

```diff
@@ -0,0 +1,17 @@
+<!DOCTYPE html>
+<html lang="en">
+
+{% block head %}
+    <head>
+        <title>CSH Packet</title>
```

Comment:
Hmmmmmm.

Reply:
We're going to pretend that wasn't there… fixed on next commit